cmd/prometheus/main.go

```diff
@@ -152,6 +153,9 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "memory-snapshot-on-shutdown":
 				c.tsdb.EnableMemorySnapshotOnShutdown = true
 				level.Info(logger).Log("msg", "Experimental memory snapshot on shutdown enabled")
+			case "extra-scrape-metrics":
+				c.scrape.EnableScrapeTimeoutMetric = true
```

Comment:
Should we rename the internal option now or when we add additional metrics?

Reply:
I was thinking later, depending on if it's actually a param to the `c.scrape` or something else.